src/qt/bitcoingui.cpp

```diff
@@ -1242,6 +1243,12 @@ void BitcoinGUI::showProgress(const QString &title, int nProgress)
     if (nProgress == 0)
     {
         progressDialog = new QProgressDialog(title, "", 0, 100);
+#ifdef Q_OS_MAC
```

Comment:
Move the implementation to something like:
```cpp
void GUIUtil::PolishProgressDialog(QProgressDialog* dialog)
{
#ifdef Q_OS_MAC
    // Workaround for macOS-only Qt bug; see: QTBUG-65750, QTBUG-70357.
    const int margin = (dialog->fontMetrics()).width("X");
    progressDialog->resize(dialog->width() + (2 * margin), progressDialog->height());
#else
    Q_UNUSED(dialog);
#endif
}
```

Reply:
Thank you @promag. Will do.